x/staging/posts/types/msgs.go

```diff
@@ -386,11 +382,11 @@ func (msg MsgRegisterReaction) ValidateBasic() error {
 	}
 
 	if !commons.IsURIValid(msg.Value) {
-		return sdkerrors.Wrap(commonerrors.ErrInvalidURI, "reaction value should be a valid uri")
+		return ErrReactionInvalidValue
```

Comment:
I think it's better to use a more generic `ErrInvalidReaction` and specify the error here